docs/Users_Guide/tc-diag.rst

```diff
@@ -226,43 +237,38 @@ If true, all input fields are read efficiently from each file in a single call.
 
-.. note:: As of MET version 11.1.0, **nc_rng_azi_flag** is the only supported output type. These configuration options will automatically be reset at runtime to the settings listed above.
+These three flag entries are booleans specifying what output data types should be written. At least one of these flags must be set to true.
 
-Configuring MET Version, Output Prefix, and Temp Directory
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  - The **nc_cyl_grid_flag** entry controls the writing of a NetCDF file containing the cylindrical coordinate range-azimuth data used to compute the diagnostics. These files are written with a `_cyl_grid_{domain}.nc` suffix, where `{domain}` is the domain name specified in the configuration file. One output file is written for each combination of model track and domain.
+  - The **nc_diag_file** entry controls the writing of the computed diagnostics to a NetCDF file. These files are written with a `_diag.nc` suffix. One output file is written for each model track processed.
+  - The **cira_diag_flage** entry controls the writing of the computed diagnostics to a formatted ASCII output file. These files are written with a `_diag.dat` suffix. One output file is written for each model track processed.
 
 .. code-block:: none
 
-  tmp_dir       = "/tmp";
-  output_prefix = "";
-  version       = "V11.1.0";
+  output_base_format = "s{storm_id}_{model}_doper_{init_time}";
 
-These options are common to multiple MET tools and are described in :numref:`config_options`.
+The **output_base_format** entry is a string that defines the naming convention that should be used when writing the output files described above. The following keywords are supported and will be replaced with values from the corresponding track: {storm_id}, {basin}, {cyclone}, {storm_name}, {technique_number}, {technique}, {init_ymdh}, {init_ymd_hms}, {init_hour}.
 
 tc_diag Output
 --------------
 
 The TC-Diag tool writes up to three output data types, as specified by flags in the configuration file. Each time TC-Diag is run it processes track data for a single initialization time. The actual number of output files varies depending on the number of model tracks provided.
 
-.. note:: As of MET version 11.1.0, **nc_rng_azi_flag** is the only supported output type.
-
 **CIRA Diagnostics Output**
 
 When the **cira_diag_flag** configuration entry is set to true, an ASCII CIRA diagnostics output file is written for each model track provided.
 
-Details will be provided when support for this output type is added.
+TODO: Details will be provided when support for this output type is added.
 
 **NetCDF Diagnostics Output**
 
 When the **nc_diag_flag** configuration entry is set to true, a NetCDF output file containing the computed diagnostics is written for each model track provided.
 
-Details will be provided when support for this output type is added.
+TODO: Details will be provided when support for this output type is added.
```

Comment:
Likewise, I thought output is not supported for this type.

Reply:
I'll update to point this #2729 which includes a note to add the documentation.